doc/qbk/3.4.path.qbk

```diff
@@ -147,30 +147,80 @@ clearly not intended. This table shows a URL string, a
 modification operation, and the URL string which results from
 applying the operation:
 
+[/
+    Functions which may need
+    to alter the path prefix:
+
+    * convert ':' to %3A
+        remove_scheme()
+
+    * add "/."
+        remove_authority()
+
+    * convert ':' to %3A or add "/."
+        remove_origin()
+
+    * convert %3A to ':'
+        set_scheme_impl()
+            set_scheme()
+            set_scheme_id()
+
+    * convert ':' to %3A
+    * convert %3A to ':'
+        set_path_absolute()
+
+    * convert ':' to %3A or add "/."
+        set_path()
+
+    ????
+        set_encoded_path()
+        edit_segments()
+]
+
 [table Path Operations [
     [URL]
     [Operation]
     [Result]
 ][
-    [`""`]
-    [`set_path( "kyle:xy" )`]
-    [`"./kyle:xy"`]
+    [`"info:kyle:xy"`]
```

Comment:
Oh, OK. We're just going to keep the examples in the end and the comment is to help us write the examples. Now I understand why you found so many bugs with the path tweaks lately.

Reply:
LMAO... yep.